README.md

```diff
@@ -1,4 +1,4 @@
-# Fast Downward
+<img src="misc/images/fast-downward.png" width="800" alt="Fast Downward">
 
```

Comment:
I was just wondering, if we want to use the SVG image on the github page. It seems to be possible (https://stackoverflow.com/questions/13808020/include-an-svg-hosted-on-github-in-markdown). Have you tried it?

I like vector graphics, as they are always sharp and this prevents the issue that people will just download the png from the github page instead of downloading the svg under misc/images/... (and most times, I think it is preferable to use the SVG file and even MS PowerPoint seems to suport SVG now).

Reply:
I tried it and the drill was drilling the right end of the 'O'. Afterwards, I simplified the svg a bit and at least it now shows up ok if I open it in the browser. Let's give it another try.

Reply:
It is so sharp now, I zoomed in and cut myself on the edge of the drill.